VoodooI2CHID/VoodooI2CTouchscreenHIDEventDriver.cpp

```diff
@@ -221,6 +221,21 @@ void VoodooI2CTouchscreenHIDEventDriver::forwardReport(VoodooI2CMultitouchEvent
         if (!event.contact_count)
             return;
     
+        //# Resolve an edgecase where if a multitouch event happens then the fact that the latest event is a stylus event
+        //# is ignored until you perform some touch that resets the contact_count to 1
+        //# we check if the event is a stylus event and prioritize that over the multitouch
+        //# This edgecase is encounterd semi-frequently, I believe it's due to bad palm rejection
+        Boolean isStylus = false;
+        for (int index = 0, count = event.transducers->getCount(); index < count; index++) {
```

Comment:
I am being pedantic here, but for consistency with the rest of the code base, please also use `i` instead of `index`.